x/dex/keeper/msg_server_limit_test.go

```diff
@@ -110,7 +110,7 @@ func (s *MsgServerTestSuite) TestMultiTickLimitOrder1to0WithWithdraw() {
 	s.aliceLimitSells("TokenB", -1, 25000)
 	s.aliceLimitSells("TokenB", 0, 25000)
 	s.aliceLimitSells("TokenB", 1, 25000)
-	s.bobMarketSells("TokenA", 40000, 30000)
+	s.bobMarketSells("TokenA", 40000, 30000, sdk.ZeroDec())
```

Comment:
Let's not add this limitPrice parameter to the `XyzMarketSells()` shorthand if it's mostly not used. A market sell with a limit price is really a limit sell, so that might be better verbiage.